portfolio/src/main/webapp/script.js

```diff
@@ -93,14 +93,41 @@ function drawChart() {
   ]);
   const options = {
     'title': 'Zoo Animals',
+    'pieHole': 0.4,
     'width':500,
     'height':400
   };
-  const chart = new google.visualization.PieChart(document.getElementById('chart-container'));
+  const chart = new google.visualization.PieChart(document.getElementById('chart-div'));
   chart.draw(data, options);
 }
 
+function drawChart2() {
```

Comment:
It might be better to change these function names to something more descriptive like `drawStockChart` and `drawAnimalChart`

Reply:
addressed in "changed function names"